tools/projmgr/schemas/cbuild-pack.schema.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://raw.githubusercontent.com/Open-CMSIS-Pack/devtools/schemas/projmgr/2.1.0/tools/projmgr/schemas/cbuild-pack.schema.json",
+  "title": "CMSIS cbuild-pack",
+  "description": "file containing all pack versions used, used for pack locking",
+  "version": "2.1.0",
+  "properties": {
+    "cbuild-pack": {
+      "$ref": "./common.schema.json#/definitions/BuildPackDescType"
+    }
+  },
+  "required": [ "cbuild-pack" ]
+}
```

Comment:
As recently requested in https://github.com/Open-CMSIS-Pack/devtools/issues/1211 please add the type definition (e.g. `"type": "object"`) and possibly the recommended `"additionalProperties": false`.